libcontainer/nsenter/nsenter_test.go

```diff
@@ -173,8 +173,10 @@ func TestNsenterChildLogging(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create log pipe %v", err)
 	}
-	defer logread.Close()
-	defer logwrite.Close()
+	defer func() {
+		_ = logread.Close()
+		_ = logwrite.Close()
+	}()
```

Comment:
Oh! I need to reverse these to keep the old behaviour; defers are triggered in the reverse order that they were added
```suggestion
	defer func() {
		_ = logwrite.Close()
		_ = logread.Close()
	}()
```

Reply:
👍 updated